antarest/core/config.py

```diff
@@ -232,6 +232,27 @@ def from_dict(data: JSON) -> "EventBusConfig":
         )
 
 
+@dataclass(frozen=True)
+class CacheConfig:
+    """
+    Sub config object dedicated to cache module
+    """
+
+    redis: Optional[RedisConfig] = None
```

Comment:
You can extract the redis config (and remove it from the EventBus config) to be a top level config (modify the example config too).
Even if EventBuseConfig is empty, keep it for now.